docs/api/boligsameie.md

```diff
@@ -612,9 +612,45 @@ feltene.
 </TabItem>
 <TabItem headerText="Informasjonsmodell" itemKey="itemKey-4">
 
-Her kan du legge inn skjermbilde av informasjonsmodell.
-
+![boligsameie](../../static/download/Informasjonsmodell_Boligsameie.png)
+
+| Eier                           | Element                              | Dokumentasjon                                                                                                                                   |
+|--------------------------------|--------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| Leveranse                      | inntektsår                           | Inntektsåret leveransen gjelder                                                                                                                 |
+| Leveranse                      | kildesystem                          | System brukt for å levere oppgaven                                                                                                              |
+| Leveranse                      | leveransetype                        | Type av leveranse som angir om leveransen inneholder ordinære oppgaver eller om oppgavegiver angir at det ikke er noen oppgaver å innrapportere |
+| Leveranse                      | oppgave                              | Oppgave som leveres                                                                                                                             |
+| Leveranse                      | oppgavegiver                         | Organisasjon som leverer oppgave på vegne av oppgaveeier                                                                                        |
+| Leveranse                      | oppgavegiversLeveranseReferanse      | Frivillig referanse på innsendingen til bruk mot egne interne systemer og evt. support mot skattetaten                                          |
+| Leveranse                      | oppgaveoppsummering                  | Oppsummering med totalsummer for innleverte oppgaver                                                                                            |
+| Leveranse                      | sletteoppgave                        | Oppgave for sletting av tidligere innrapporterte oppgaver                                                                                       |
+| Melding                        | leveranse                            | Selve leveransen. Merk at det kun er tillatt med en leveranse pr Melding                                                                        |
```

Comment:
Ble usikker nå. Hva skrev du på boligselskap her? Der kan man ha flere leveranser, men de må gjelde for samme oppgavegiver, og de må ha forskjellig boligselskapsformål (tror jeg, den siste biten her må dobbelsjekkes)

Reply:
Bra fanget.  Begrepsdefinisjonen for oppgavegiver er: https://[data.skatteetaten.no/begrep/20b52aba-9fe1-11e5-a9f8-e4115b280940](https://data.skatteetaten.no/begrep/20b52aba-9fe1-11e5-a9f8-e4115b280940)

"tredjepart som rapporterer opplysning til Skatteetaten"
De rapporterer ikke på vegne av oppgaveier.  Oppgaveeier har ikke plikt til å levere, og har vel heller ikke lov til å rapportere.

Reply:
> Ble usikker nå. Hva skrev du på boligselskap her? Der kan man ha flere leveranser, men de må gjelde for samme oppgavegiver, og de må ha forskjellig boligselskapsformål (tror jeg, den siste biten her må dobbelsjekkes)

Det står det samme for boligselskap. Forstår jeg deg rett som at vi må endre det i boligselskap da, men at det er riktig for boligsameie?

Reply:
> Bra fanget. Begrepsdefinisjonen for oppgavegiver er: https://[data.skatteetaten.no/begrep/20b52aba-9fe1-11e5-a9f8-e4115b280940](https://data.skatteetaten.no/begrep/20b52aba-9fe1-11e5-a9f8-e4115b280940)
> 
> "tredjepart som rapporterer opplysning til Skatteetaten" De rapporterer ikke på vegne av oppgaveier. Oppgaveeier har ikke plikt til å levere, og har vel heller ikke lov til å rapportere.

Da oppdaterer jeg til det for både boligsameie og -selskap

Reply:
Ja, stemmer. Den biten om antall leveranser må oppdateres for boligselskap